ci-operator/config/openshift/configuration-anomaly-detection/configuration-anomaly-detection-master.yaml

```diff
@@ -0,0 +1,39 @@
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  secret:
+    mount_path: /tmp/secret
+    name: configuration-anomaly-detection-codecov-token
+- as: publish-coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  postsubmit: true
+  secret:
+    mount_path: /tmp/secret
+    name: configuration-anomaly-detection-codecov-token
+- as: lint
+  commands: make lint
+  container:
+    from: src
+- as: test
+  commands: make test
+  container:
+    from: src
+- as: validate
+  commands: make validate
+  container:
+    from: src
```

Comment:
I added the `make ci-check` to hold the right checks for this..
do you think you could change the `ci-check` to do these things?

Reply:
I had to modify our Makefile (https://github.com/openshift/configuration-anomaly-detection/pull/27/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52)

It **must** have the `coverage` PHONY according to the SOP... I guess this is related to Line 21 of this PR. We can remove the other sub tests.. or we put golangci lint in this as well ( right now golangci would be triggered via Github Actions).

Reply:
yeah, I think adding `coverage` into `make ci-check` is good and I would want this to be the only entrypoint for CI

this way we have `make` for common dev and `make ci-check` for ci check

we might want to to add `make containerized-ci-check` later on if we want to check why it failed on the CI server